xbmc/guilib/TexturePi.h

```diff
@@ -8,26 +8,23 @@
 
 #pragma once
 
-#include "TextureGL.h"
+#include "TextureGLBase.h"
 
 #include "system_gl.h"
 
-/************************************************************************/
-/*    CGLTexture                                                       */
-/************************************************************************/
-class CPiTexture : public CGLTexture
+class CPiTexture : public CTextureGLBase
 {
 public:
   CPiTexture(unsigned int width = 0, unsigned int height = 0, unsigned int format = XB_FMT_A8R8G8B8);
   virtual ~CPiTexture();
-  void CreateTextureObject();
-  void LoadToGPU();
+  void CreateTextureObject() override;
+  void LoadToGPU() override;
   void Update(unsigned int width, unsigned int height, unsigned int pitch, unsigned int format, const unsigned char *pixels, bool loadToGPU);
   void Allocate(unsigned int width, unsigned int height, unsigned int format);
   bool LoadFromFileInternal(const std::string& texturePath, unsigned int maxWidth, unsigned int maxHeight, bool requirePixels, const std::string& strMimeType = "");
 
-protected:
-
 private:
   void *m_egl_image;
 };
+
+using CTexture = CPiTexture;
```

Comment:
This conflicts with `using CTexture = CTextureGLES;` in `xbmc/guilib/TextureGLES.h`